src/input.service.ts

```diff
@@ -278,21 +336,23 @@ export class InputService {
   private subscriptions: Subscription[] = [];
   private pollRaf: number = null;
 
-  constructor(private focus: FocusService) {}
+  constructor(private focus: FocusService) { }
 
   /**
    * Bootstrap attaches event listeners from the service to the DOM and sets
    * up the focuser rooted in the target element.
    */
-  public bootstrap(root: HTMLElement = document.body) {
+  public bootstrap(handleTab : (direction : Direction) => any = (direction : Direction) => { direction }, root: HTMLElement = document.body) {
+    this.handleTab = handleTab;
```

Comment:
Could you explain what you mean by "Consumers can listen to the gamepad themselves if they need a global handler"? The input service is polling from the gamepad, so shouldn't all listening to the gamepad be done there?